docs_nnx/nnx_basics.ipynb

```diff
@@ -77,14 +90,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "`nnx.Variable`'s inner values can be accessed using the `.value` property, however\n",
-    "for convenience they implement all numeric operators and can be used directly in\n",
-    "arithmetic expressions (as shown above).\n",
+    "Also note that:\n",
+    "\n",
+    "- The inner values of [`nnx.Variable`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/variables.html#flax.nnx.Variable) can be accessed using the `value` property, but for convenience they implement all numeric operators and can be used directly in arithmetic expressions (as shown in the code above).\n",
+    "- Additionally, [`nnx.Variable`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/variables.html#flax.nnx.Variable)s can be passed to any JAX function because they implement the `__jax_array__` protocol (as long as their inner value is a JAX array).\n",
```

Comment:
https://github.com/google/flax/pull/4236 removed the mention of `__jax_array__` recently, so we should apply its change here too.

Reply:
Done!